cogs/greetings.py

```diff
@@ -3,22 +3,26 @@
 import os
 from discord.ext import commands
 
+# Class dedicated to greeting new members when joining the server
 class Greetings(commands.Cog):
+
+# Constructor that assigns the welcome message to the specific new member with their username
     def __init__(self, client):
         self.client = client
         self.welcome_message_id = self.load_welcome_message_id()
-
+# Helper function responsible for loading and returning the welcome message
```

Comment:
Would like some spacing in between our functions. This is for the whole file

Reply:
Done